internal/apis/config/webhook/types.go

```diff
@@ -111,7 +111,7 @@ type DynamicServingConfig struct {
 	// used as a CA to sign dynamic serving certificates.
 	SecretNamespace string
 
-	// Namespace of the Kubernetes Secret resource containing the TLS certificate
+	// Kubernetes Secret resource containing the TLS certificate
```

Comment:
Can you update this sentence so it matches the other descriptions exactly?
```suggestion
	// Secret resource name containing the TLS certificate
```